test/lsp/lsp_hover.ml

```diff
@@ -143,3 +167,879 @@ let%expect_test "hover-replaced" =
     ```cobol
     "B" "C"
     ``` |}];;
+
+(* Hover typedef vars *)
+
+let%expect_test "hover-typedef-vars" =
+  let { projdir; end_with_postproc }, server = make_lsp_project () in
+  print_hovered server ~projdir @@ extract_position_markers {cobol|
+        IDENTIFICATION DIVISION.
+        PROGRAM-ID. prog.
+        DATA DIVISION.
+        WORKING-STORAGE SECTION.
+        01 DATA-N_|_AME PI_|_C X.
+        01 SAME-LIN_|_E-1 PIC X. 01 S_|_AME-LINE-2 PIC 9.
```

Comment:
Did you have issues with these same-line cases? (I'm wondering why you needed to add this test.)

Reply:
It was needed when I retrieved the pic string directly from the file !